<launch>
  <arg name="ros_hz" default="7.0"/>
  <arg name="os_display" default="true"/>

  <arg name="face_frame_id" default="face_frame"/>
  <arg name="focus_frame_id" default="focus_frame"/>
  <arg name="camera_frame_id" default="zed2_camera_center"/>
  <arg name="world_frame_id" default="world"/>
  
  <arg name="machine_name" default="tobojetuk"/>
  
  <param name="zed_resolution" value="1.0"/>

  <param name="configs_path" value="/home/$(arg machine_name)/catkin_ws/src/tobo_perception"/>
  <param name="recordings_path" value="/media/$(arg machine_name)/Maxtor/recording_test/"/>
  
  <param name="os_display" value="$(arg os_display)"/>
  <param name="ros_hz" value="$(arg ros_hz)"/>
  <param name="focus_frame_id" value="$(arg focus_frame_id)"/>
  
  <!-- Static transforms Enviroment setup -->
  <include file="$(find tobo_perception)/launch/scenario_calib.launch"/>

  <node pkg="tobo_perception" type="perceptionstream_node" name="perceptionstream_node" output="screen">
  </node>
  
  <node pkg="tobo_perception" type="focus_estimation" name="focus_estimation_node" output="screen">
  </node>
  
  <!--node type="rviz" name="rviz" pkg="rviz" args="-d $(find tobo_perception)/configs/perception.rviz"/-->
</launch>

<!-- roslaunch tobo_perception tobo_perception.launch os_display:=true ros_hz:=10.0 src_zed_gst:=true-->
